nuxt/components/form/Form.vue

```diff
@@ -61,6 +68,26 @@ const props = withDefaults(defineProps<Props>(), {
   submitName: undefined,
 })
 
+const store = useMaevsiStore()
+const turnstileKey = ref('')
+const disabledSubmitButton = ref(true)
```

Comment:
Booleans variables should be prefixed by `is` for consistency.
```suggestion
const isSubmitButtonDisabled = ref(true)
```

Reply:
I've changed the name to what I initially recommended (89e79f75f02b73984f6c1abf0b9cd1fb8af52d62).